powerauth-java-server/src/test/java/io/getlime/security/powerauth/app/server/database/repository/OperationRepositoryTest.java

```diff
@@ -0,0 +1,86 @@
+package io.getlime.security.powerauth.app.server.database.repository;
+
+import static io.getlime.security.powerauth.app.server.database.model.OperationStatusDo.EXPIRED;
+import static io.getlime.security.powerauth.app.server.database.model.OperationStatusDo.PENDING;
+import static io.getlime.security.powerauth.crypto.lib.enums.PowerAuthSignatureTypes.BIOMETRY;
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.util.List;
+
+import org.joda.time.DateTime;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.transaction.annotation.Transactional;
+
+import io.getlime.security.powerauth.app.server.database.model.entity.ApplicationEntity;
+import io.getlime.security.powerauth.app.server.database.model.entity.OperationEntity;
+import io.getlime.security.powerauth.crypto.lib.enums.PowerAuthSignatureTypes;
+
+/**
+ * Test for {@link OperationRepository} CRUD operations.
+ */
+@DataJpaTest
+public class OperationRepositoryTest {
+
+    @Autowired
```

Comment:
Can we switch to constructor or setter autowiring? Field injection is not recommended, see e.g.:
https://cloudolife.com/2021/02/27/Programming-Language/Java/FAQs/Field-injection-is-not-recommended-and-Injection-guidelines-in-Java-Spring/